services/drupal/web/modules/custom/view_mode_select_by_token/src/Plugin/Field/FieldFormatter/RenderedWithSelect.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace Drupal\view_mode_select_by_token\Plugin\Field\FieldFormatter;
+
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\Core\Field\Plugin\Field\FieldFormatter\EntityReferenceEntityFormatter;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\entity_browser\Plugin\EntityBrowser\FieldWidgetDisplay\RenderedEntity;
+
+
+/**
+ * Plugin implementation of the 'Render entity to Markdown' formatter.
+ *
+ * @FieldFormatter(
+ *   id = "view_mode_select_by_token_rendered_with_select",
+ *   label = @Translation("Rendered With Select"),
```

Comment:
I'm wondering if we can come up with another label for this. "Rendered With Select" doesn't really mean much to me. Maybe something like "Render with variable view mode", or "Render with dynamic view mode"?  Any thoughts on that? Not the biggest deal, just trying to make this be something the general public will understand.